src/Lister.php

```diff
@@ -45,7 +45,7 @@ public function renderView()
             }
 
             $this->t_row->trySet('_title', $this->model[$this->model->title_field]);
-            $this->t_row->trySet('_href', $this->app->url(['id'=>$this->current_id]));
+            $this->t_row->trySet('_href', $this->url(['id'=>$this->current_id]));
```

Comment:
need to test this change

Reply:
Tested locally with:

``` php
    $frame->add(['Lister', "defaultTemplate"=>"lister.html"])->setSource(['foo','bar','baz']);
```

so works fine, but no good demo for this.